umm-spec-lib/test/cmr/umm_spec/test/generate_and_parse.clj

```diff
@@ -157,13 +161,19 @@
              (format "Parsing example file %s and converting to %s and then parsing again did not result in expected umm."
                      example-file target-format)))))))
 
-(deftest roundtrip-example-collection-record
-  (doseq [metadata-format tested-collection-formats]
+(deftest roundtrip-example-collection-record-non-iso19115
```

Comment:
Why were the ISO tests separated out - the code is the same?

Reply:
It's separated out because I need to test the case when UMM-C contains Footprints and doesn't contain SwathWidth. And that only works for iso19115. For echo10 and dif10, you have to manually remove the Footprints, convert the largest footprint to SwathWidth, it just looks messy.

Reply:
Modified the test.

Reply:
But I don't see what has changed other than it was separated out.